### PR TITLE
feat(strm-1230): expose adding, removing and listing projects members

### PR DESCRIPTION
**Pull request by bobvandenhoogen**

### CODE REVIEW

File: pkg/entity/project/project.go [hunk 1]
```diff
@@ -16,7 +18,10 @@ var client projects.ProjectsServiceClient
 var apiContext context.Context
 
 const (
-	descriptionFlag = "description"
+	descriptionFlag       = "description"
+	addMemberFlag         = "add-member"
+	removeMemberFlag      = "remove-member"
```
Comment (ivan-p92) [inline pkg/entity/project/project.go:23]: perhaps the flag should be plural, since we can add multiple?

File: pkg/entity/project/project.go [hunk 2]
```diff
@@ -56,3 +61,32 @@ func create(projectName *string, cmd *cobra.Command) *projects.CreateProjectResp
...
+	}
+
+	_, err = client.AddProjectMembers(apiContext, addReq)
+	common.CliExit(err)
+	_, err = client.RemoveProjectMembers(apiContext, removeReq)
+	common.CliExit(err)
+	return
+}
+
+func GetActiveProject() string {
```
Comment (ivan-p92) [inline pkg/entity/project/project.go:87]: You could use `GetActiveProject` from the context package, I think?

File: pkg/entity/project/cmd.go [hunk 3]
```diff
@@ -33,3 +33,22 @@ func CreateCmd() *cobra.Command {
 	flags.String(descriptionFlag, "", "description of the project")
 	return project
 }
+
+func ManageCmd() *cobra.Command {
+	project := &cobra.Command{
+		Use:   "project",
+		Short: "Manage a project",
```
Comment (ivan-p92) [inline pkg/entity/project/cmd.go:40]: Should we add a more elaborate description?

**Review by ivan-p92 — CHANGES_REQUESTED** [review]

File: pkg/entity/project/project.go (hunk 2 above)
Comment (bobvandenhoogen) [reply pkg/entity/project/project.go:87]: It didn't import. That's why I copy-pasted it

File: pkg/entity/project/project.go (hunk 1 above)
Comment (bobvandenhoogen) [reply pkg/entity/project/project.go:23]: good call

File: pkg/entity/project/cmd.go (hunk 3 above)
Comment (bobvandenhoogen) [reply pkg/entity/project/cmd.go:40]: yes, forgot about this

File: go.mod [hunk 4]
```diff
@@ -19,11 +19,13 @@ require (
...
+	github.com/strmprivacy/api-definitions-go/v2 v2.48.0
 	golang.org/x/oauth2 v0.0.0-20211104180415-d3ed0bb246c8
 	golang.org/x/sync v0.0.0-20210220032951-036812b2e83c
 	google.golang.org/grpc v1.46.0
 	google.golang.org/protobuf v1.27.1
 	gopkg.in/natefinch/lumberjack.v2 v2.0.0
 	sigs.k8s.io/yaml v1.3.0
 )
+
+replace github.com/strmprivacy/api-definitions-go/v2 => ../api-definitions/lang/go/build
```
Comment (bobvandenhoogen) [inline go.mod:31]: TODO

**Review by bvdeenen — APPROVED** [review]

**Review by ivan-p92 — APPROVED** [review]

**Pull request merged**